.travis.yml

```diff
@@ -13,6 +13,10 @@ matrix:
       env: SYMFONY_VERSION='2.7.*'
     - php: 5.6
       env: SYMFONY_VERSION='2.8.*'
+    - php: 7.0
+      env: SYMFONY_VERSION='4.0.*'
+    - php: 7.1
+      env: SYMFONY_VERSION='4.0.*'
```

Comment:
We only add LTS versions in this list, and 4.0 is not. And we don't have the same version against multiple PHP versions.

What should be done to test against Symfony 4.0 is to change the `DEPENDENCIES='dev'` to run on PHP 7.1 rather than 5.6